frontend/sozisel-app/src/components/TemplatesList/TemplateList.tsx

```diff
@@ -77,7 +83,7 @@ export default function TemplateList(): ReactElement {
     } catch (error) {
       console.error(error);
     }
-  };
+  }, []);
```

Comment:
delete mutation and clone mutation above should be a dependency inside useCallback